### ISSUE TITLE
There is no way to share manifest snapshots links

### ISSUE THREAD

**Issue by Symbianx** [nosinovacao/name-registry-ui#5]
There is currently no way to send a manifest snapshot to someone via link, this should be possible in order to allow collaboration when diagnosing problems.

## Changes:
Add the currently selected manifest snapshot and session to the url, for easy copying. With a result as:
`http://192.168.1.1/service/7D6A5C76D1B45B6E965A96C8BE1AAC19?sessionId=b0fe13be-3dd1-47af-bf92-8509c94e3e2e&manifestSnapshotFrom=2017-08-01T13%3A26%3A10.791%2B00%3A00`

**Comment by suvl** [comment]
The very example you present reveals a weakness in logic: inaccessible servers. While I am not against this feature (it will be extremely useful for the folks that ultimately manage the production infrastructures 'round the world), we must consider the "export" feature, where we export (a) (human-readable) report(s) for the entire session(s).

**Comment by Symbianx** [comment]
I agree that an export feature is needed as well, but I think we should consider both, as they both have their utility.
@suvl consider creating a different issue for the export feature.
